README.md

```diff
@@ -62,6 +62,16 @@ Then you can periodically run the `download_authenticator_metadata` management c
 If metadata are available for the given `Authenticator`, its `metadata` property will be an object.
 The `level`, `vulnerabilities` and `is_update_available` methods on `metadata` can be used to determine the trust and certification level.
 
+## Passwordless
+
+This authentication requires "discoverable credential" and using that credential to perform a user lookup using the passwordless authentication backend
+
+1. Set `DJANGO_FIDO_RESIDENT_KEY` to `True`
```

Comment:
This could probably be handled and checked during application setup to have some sensible combination.

Reply:
Where is the best place to put that logic in?

Reply:
I presume `SETTINGS.check()`?

Reply:
Yes, that is probably the best place.

Reply:
Please see my last commit for the added settings check against passwordless auth.

Side note: My Django 3.2 doesn't load the `DjangoFidoConfig` automatically by just defining `django-fido` in the `INSTALLED_APPS`, for some reason I had to define `django_fido.apps.DjangoFidoConfig` to make it load.